_posts/2021-04-07-upgrade-improvements-and-migration.md

```diff
@@ -0,0 +1,129 @@
+---
+layout: post
+title:  "Upgrade changes and API versions conversion"
+date: 2021-04-07
+author: jakub_stejskal
+---
+Upgrade process of Strimzi and Kafka is not trivial. 
+In Strimzi 0.22 we introduced several improvements, which will make the process easier for the users. 
+Moreover, we also introduced new API version `v1beta2` which will replace `v1beta1` because [Kubernetes 1.22 will remove support](https://kubernetes.io/docs/reference/using-api/deprecation-guide/#customresourcedefinition-v122) for `apiextensions/v1beta1`.
```

Comment:
I do not think people will understand this. You might need to explain more ... but that would make it too long for the perex. so maybe we could say here just something like ...

```suggestion
Moreover, we also introduced new API version `v1beta2` which will replace `v1beta1` / `v1alpha1` APIs and make Strimzi ready for Kubernetes 1.22.
```

... and leave the details for the main text?